src/utils/PeopleWithParents.ts

```diff
@@ -0,0 +1,27 @@
+import { Person } from '../types';
+
+export const peopleWithPerents = (peopleFromServer: Person[]): Person[] => {
```

Comment:
There is a typo in the function name `peopleWithPerents`. It should be corrected to `peopleWithParents` to match the intended meaning and maintain consistency with the naming conventions.